lib/executor/constant_arrival_rate.go

```diff
@@ -164,6 +164,11 @@ func (carc ConstantArrivalRateConfig) NewExecutor(
 	}, nil
 }
 
+// HasWork reports whether there is any works for give execution segment.
```

Comment:
```suggestion
// HasWork reports whether there is any work to be done for the given execution segment.
```
same fix for the other copies of the method

Reply:
Fixed